django/contrib/gis/db/backends/oracle/adapter.py

```diff
@@ -15,6 +15,7 @@ def __init__(self, geom):
          * Outer ring - counter clockwise
          * Inner ring(s) - clockwise
         """
+        geom = geom.clone()
```

Comment:
We shouldn't clone unless it has to be fixed or at least we shouldn't clone unless it's a `Polygon` or `GeometryCollection` (because of performance).

Generally, it might make sense to deprecate fixing polygons for Oracle because:
1. it breaks basic assumption that what we read is the same what we have just written
2. our implementation seems to be quite slow and in some scenarios polygons could be fixed before the get into Django